fly/commands/pause_job.go

```diff
@@ -22,16 +24,20 @@ func (command *PauseJobCommand) Execute(args []string) error {
 		return err
 	}
 
-	found, err := target.Team().PauseJob(command.Job.PipelineName, command.Job.JobName)
+	team := command.TeamFlag.GetTeamTarget(target)
+	found, err := team.PauseJob(pipelineName, jobName)
 	if err != nil {
 		return err
 	}
 
 	if !found {
-		return fmt.Errorf("%s/%s not found\n", command.Job.PipelineName, command.Job.JobName)
+		if command.Team == "" {
+			fmt.Println("hint: are you missing '--team' to specify the team for the build?")
+		}
+		return fmt.Errorf("%s/%s not found\n", pipelineName, jobName)
```

Comment:
how about just amending this to say 'not found in team X'? rather than guessing with a hint maybe they'll notice the problem in the message